### ISSUE TITLE
Add RTL support

### ISSUE THREAD

**Issue by limonte** [limonte/sweetalert2#707]
- ~close button should be placed in top-left corner~ (done in #774)
- for the `position` parameter there not be `-start` and `-end` aliases for `-left` and `-right`

**Comment by limonte** [comment]
>  for the `position` parameter there should not be `left` or `right`. Instead, align with flex - `start` `end`

This is going to be a breaking change in the next major.

**Comment by toverux** [comment]
> This is going to be the breaking change in the next major.

Start/end could be added as aliases for left/right (and remove left/right from the documentation) - so the next major will "break less".

**Comment by limonte** [comment]
> Start/end could be added as aliases for left/right (and remove left/right from the documentation) - so the next major will "break less".

Good thinking, aliases will be the additional feature for the next minor, major isn't required at all.
